wire-ios-request-strategy/Sources/APIs/MessageAPI.swift

```diff
@@ -265,6 +271,87 @@ class MessageAPIV4: MessageAPIV3 {
     override var apiVersion: APIVersion {
         .v4
     }
+
+    private let protobufContentType = "application/x-protobuf"
+
+    override func broadcastProteusMessage(message: any ProteusMessage) async throws -> (Payload.MessageSendingStatus, ZMTransportResponse) {
+        let path = "/broadcast/proteus/messages"
```

Comment:
The path has changed from the current implementation (/broadcast/otr/messages). Just a sanity check that we want this.

Reply:
Yes the path changed from v0 to v1.

Reply:
Oh yes I see. I was looking at v0 not v1 implementation.